### ISSUE TITLE
Hiscores aren't working?

### ISSUE THREAD

**Issue by VenomousInc** [michaelbull/rs-api#2]
I'm attempting to pull from the hiscores, but it just returns as **_Optional.empty_**
```
        String username = "le me";
        StringBuilder sb = new StringBuilder("Test Command -> ");

        RuneScapeAPI rsa = RuneScapeAPI.createHttp();
        Hiscores hiscores = rsa.hiscores();

        try {
            if(hiscores.playerInformation(username, HiscoreTable.DEFAULT).isPresent()) {
                sb.append(hiscores.playerInformation(username, HiscoreTable.DEFAULT).toString());
            } else {
                sb.append("Can't pull: " + hiscores.playerInformation(username, HiscoreTable.DEFAULT));
            }
        } catch (IOException e) {
            e.printStackTrace();
        }

        System.out.println(sb);
```

**Comment by michaelbull** [comment]
Thanks for the bug report, seems like the issue is the new hiscore activities that have been added. I'll push a fix soon.

**Comment by michaelbull** [comment]
Fixed in v1.1.0
